### PR TITLE
IDAM-511 Bump CRA version and add path aliases

### PR DESCRIPTION
**Pull request by johnholland-nice**

### CODE REVIEW

File: NICE.Identity.Management/ClientApp/src/views/EditOrganisationUsers/__tests__/EditOrganisationUsers.test.js [hunk 1]
```diff
@@ -1,199 +1,288 @@
...
+  await act(async () => {
+		userEvent.paste("kri")
+		jest.advanceTimersByTime(1000);	
+  });
+  let suggestion = await screen.findAllByText("Kristin Patrick");
+  expect(suggestion.length).toEqual(1);
+  userEvent.tab();
+  suggestion = screen.queryAllByText("Kristin Patrick");
+  expect(suggestion.length).toEqual(0);
+  jest.useFakeTimers();
```
[the remark this answers is not in the document]
Comment (johnholland-nice) [reply NICE.Identity.Management/ClientApp/src/views/EditOrganisationUsers/__tests__/EditOrganisationUsers.test.js:246]: Ouch. Seem to have done this in a few places. It is meant to be useRealTimers.

File: NICE.Identity.Management/ClientApp/src/components/ResultsList/ResultsList.tsx [hunk 2]
```diff
@@ -6,12 +6,12 @@ type ResultsListProps<T> = {
 	qaSelExtract: string;
 };
 
-export const ResultsList = <T extends unknown>({
+export const ResultsList = <T,>({
```
[the remark this answers is not in the document]
Comment (johnholland-nice) [reply NICE.Identity.Management/ClientApp/src/components/ResultsList/ResultsList.tsx:9]: ResultsList can have multiple types - I think. I had to add the comma as the newer version of ts doesn't like `extend unknown`. If I change it to just <T>, it then thinks it's a jsx element for some reason. I've put the new rule and the answer I found about the trailing comma below:

https://typescript-eslint.io/rules/no-unnecessary-type-constraint/
https://stackoverflow.com/questions/32696475/typescript-tsx-and-generic-parameters

File: NICE.Identity.Management/ClientApp/src/components/ResultOrganisation/ResultOrganisation.tsx [hunk 3]
```diff
@@ -2,7 +2,7 @@ import React, { FC } from "react";
 import { Link } from "react-router-dom";
 import Moment from "moment";
 import { Card } from "@nice-digital/nds-card";
-import { OrganisationType } from "../../models/types";
+import type { OrganisationType } from "src/models/types";
```
Comment (johndavey72) [inline NICE.Identity.Management/ClientApp/src/components/ResultOrganisation/ResultOrganisation.tsx:5]: I think 'type' goes inside the curly braces - not sure if it makes a difference for a single named import like this but would be clearer to read.... import {type OrganisationType} from "..."

File: NICE.Identity.Management/ClientApp/src/components/SuggestionUser/SuggestionUser.tsx [hunk 4]
```diff
@@ -1,5 +1,5 @@
 import React, { FC } from "react";
-import { UserType } from "../../models/types";
+import type { UserType } from "src/models/types";
```
Comment (johndavey72) [inline NICE.Identity.Management/ClientApp/src/components/SuggestionUser/SuggestionUser.tsx:2]: see prev comment

File: NICE.Identity.Management/ClientApp/src/views/EditOrganisationUsers/EditOrganisationUsers.tsx [hunk 5]
```diff
@@ -5,14 +5,14 @@ import { Breadcrumbs, Breadcrumb } from "@nice-digital/nds-breadcrumbs";
...
-import { ErrorMessage } from "../../components/ErrorMessage/ErrorMessage";
-import { UsersAndJobIdsByOrganisationType, UserType } from "../../models/types";
-import { UserStatus } from "../../components/UserStatus/UserStatus";
-import { FilterSuggestions } from "../../components/FilterSuggestions/FilterSuggestions";
-import { SuggestionUser } from "../../components/SuggestionUser/SuggestionUser";
+import { Endpoints } from "src/data/endpoints";
+import { fetchData } from "src/helpers/fetchData";
+import { isDataError } from "src/helpers/isDataError";
+import { ErrorMessage } from "src/components/ErrorMessage/ErrorMessage";
+import type { UsersAndJobIdsByOrganisationType, UserType } from "src/models/types";
```
Comment (johndavey72) [inline NICE.Identity.Management/ClientApp/src/views/EditOrganisationUsers/EditOrganisationUsers.tsx:12]: not sure about best practice here, {type UsersAndJobIds... , type UserType} perhaps?

**Review by johndavey72 — APPROVED** [review]

**Pull request merged**